app/src/main/java/com/mashup/lastgarden/ui/scent/ScentViewModel.kt

```diff
@@ -0,0 +1,88 @@
+package com.mashup.lastgarden.ui.scent
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.mashup.lastgarden.data.vo.ScentItem
+
+class ScentViewModel : ViewModel() {
+
+    private val _scentList = MutableLiveData<List<ScentItem>>()
+    val scentList: LiveData<List<ScentItem>>
+        get() = _scentList
+
+    var _position = MutableLiveData<Int>()
```

Comment:
아래처럼 변경하는게 맞을까요? 변수 이름이 약간 애매한것 같아요!
```suggestion
    private val _position = MutableLiveData<Int>()
```

Reply:
setter로 변경했습니다!